src/platform/user/profile/vet360/containers/VAPProfileField.jsx

```diff
@@ -0,0 +1,366 @@
+import React from 'react';
```

Comment:
Based heavily on the existing `Vet360ProfileField` component. This is the core component used by each of the pieces of contact info shown on the the Profile. Because modals are no longer used, this component's render method is responsible for deciding which view to render:
- empty data (read-only view when contact info has not been set)
- read-only view when data is set
- edit-view
- data validation view

The old version was always showing the read-only view; it just layered modals on top of it as needed.